plugins/modules/web_infrastructure/jenkins_plugin.py

```diff
@@ -429,12 +429,12 @@ def install(self):
                 self.module.fail_json(
                     msg="Jenkins home directory doesn't exist.")
 
-            md5sum_old = None
+            sha1sum_old = None
             if os.path.isfile(plugin_file):
                 # Make the checksum of the currently installed plugin
-                with open(plugin_file, 'rb') as md5_plugin_fh:
-                    md5_plugin_content = md5_plugin_fh.read()
-                md5sum_old = hashlib.md5(md5_plugin_content).hexdigest()
+                with open(plugin_file, 'rb') as sha1_plugin_fh:
+                    sha1_plugin_content = sha1_plugin_fh.read()
```

Comment:
How about removing the name of the hash function from these two objects? That way, the diff will be more concise when (or if) the hash needs to be changed next time :)
```suggestion
                    plugin_content = plugin_fh.read()
```

Reply:
Removed all references to sha1 in variable names for code touched in this PR. Great idea!